libfwupdplugin/fu-bluez-device.c

```diff
@@ -470,8 +471,10 @@ fu_bluez_device_parse_device_information_service(FuBluezDevice *self, GError **e
 
 	fw_revision =
 	    fu_bluez_device_read_string(self, FU_BLUEZ_DEVICE_UUID_DI_FIRMWARE_REVISION, NULL);
-	if (fw_revision != NULL)
+	if (fw_revision != NULL) {
+		fu_device_set_version_format(FU_DEVICE(self), fu_version_guess_format(fw_revision));
```

Comment:
Hm... just the note -- I've connected several BT devices and only Logitech mouse expose DI and version properly, all others either have no DI service, or do not provide the version.

For Logitech:
```
10:56:34.379 FuBluezDevice        ReadValue[00002a26-0000-1000-8000-00805f9b34fb]:52 42 4d 31 38 2e 30 31 5f 30 30 31 31 
10:56:34.379 FuDevice             changing verfmt for (null): bcd->plain
10:56:34.379 FuDevice             changing version for (null): 0.11->17
10:56:34.379 FuDevice             changing version for (null): 17->RBM18.01_0011
```

Reply:
Ahh, so in the Logitech mouse case the `fw_revision` is `17`? I think that's *fine* right?

Reply:
Yes, `fw_revision` is `52 42 4d 31 38 2e 30 31 5f 30 30 31 31` ->`RBM18.01_0011`  and it is correct value -- the same is read if connected through the dongle. That's why the log attached.

But my note was mostly about that not a lot of vendors use DI to provide F/W version :(